letsencrypt-nginx/letsencrypt_nginx/dvsni.py

```diff
@@ -92,6 +92,10 @@ def _mod_config(self, ll_addrs):
         included = False
         directive = ['include', self.challenge_conf]
         root = self.configurator.parser.loc["root"]
+
+        self.configurator.parser.add_http_directives(root,
```

Comment:
I would put this at https://github.com/letsencrypt/letsencrypt/blob/nginx-bucket-size/letsencrypt-nginx/letsencrypt_nginx/dvsni.py#L102 as
```
if bucket_directive not in body:
    body.append[bucket_directive]
```
since we are looping to find the HTTP block anyway, and this avoids duplication.